debian/copyright

```diff
@@ -0,0 +1,31 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: tvb-library
+Upstream-Contact: TVB TEAM <tvb.admin@thevirtualbrain.org>
+Source: https://github.com/the-virtual-brain/tvb-library
```

Comment:
in lines 2-4 a mistake is made, by using name 'tvb-library' instead of 'tvb-gdist'.
Pls also check other similar files in the other open PRs